source/_integrations/energyzero.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: EnergyZero
+description: Instructions on how to integrate EnergyZero within Home Assistant.
+ha_category:
+  - Energy
+ha_release: 2023.2
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@klaasnicolaas'
+ha_domain: energyzero
+ha_platforms:
+  - sensor
+ha_quality_scale: platinum
+ha_integration_type: integration
+---
+
+The EnergyZero integration integrates the [EnergyZero](https://www.energyzero.nl/) API platform with Home Assistant.
+
+The integration makes it possible to retrieve the dynamic energy/gas prices from EnergyZero in order to gain insight into the price trend of the day and to adjust your consumption accordingly.
+
+Partners who purchase their energy through EnergyZero:
+
+- [ANWB Energie](https://www.anwb.nl/huis/energie/anwb-energie)
+- [Mijndomein Energie](https://www.mijndomein.nl/energie)
+- [Energie van Ons](https://www.energie.vanons.org)
+- [GroeneStroomLokaal](https://www.groenestroomlokaal.nl)
```

Comment:
Should these be added as virtual integrations instead?

Reply:
That is ultimately the intention, but as long as they have not yet been created, I have put this list there 😄